tests.rb

```diff
@@ -1,16 +1,73 @@
 require './person'
 require './student'
-require './teacher'
-require './concrete_decorators'
+require './classroom'
+require './rental'
+require './book'
 
-person = Person.new(22, 'maximilianus')
-p person
-p person.correct_name
+p 'Student-Classroom tests'
+p pedro = Student.new(501, 19, 'Pedro Salas')
+p laura = Student.new(103, 21, 'Laura Ponce')
+p mairy = Student.new(207, 20, 'Mairy Calderón')
 
-capitalized_person = CapitalizeDecorator.new(person)
-p capitalized_person
-p capitalized_person.correct_name
+p class619 = Classroom.new(101)
+p class619.students
 
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-p capitalized_trimmed_person
-p capitalized_trimmed_person.correct_name
+p pedro.classroom
```

Comment:
You made a great effort by adding tests.rb to your project. I absolutely liked it!